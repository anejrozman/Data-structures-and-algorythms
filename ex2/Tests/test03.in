15,15,0
21.93,6.96,50.93,23.96
76.12,51.26,81.12,71.25999999999999
7.72,55.6,23.72,74.6
12.09,84.31,34.09,87.31
63.58,66.91,66.58,92.91
58.2,9.66,87.2,22.66
0,31.94,15,42.94
40.36,69.16,44.36,72.16
37.8,40.72,43.8,57.72
5.25,10.43,12.25,18.43
45.87,78.06,52.87,81.06
59.85,33.73,79.85,40.73
20.47,31.88,31.47,48.879999999999995
96,37.67,100,40.67
87,62.37,100,81.37

76.12,51.26
44.36,69.16
63.58,92.91
100,40.67
12.25,18.43
40.36,69.16
59.85,33.73
63.58,92.91
43.8,57.72
0,42.94
87.2,22.66
7.72,55.6
58.2,9.66
58.2,9.66
23.72,55.6
