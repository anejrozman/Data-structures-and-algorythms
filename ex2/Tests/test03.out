150.76947909276663
152.6421832498938
152.45541053574073
180.05454015421918
144.40467383027962
153.7577676900737
150.84435645915465
152.45541053574073
148.87256103396118
179.6527019065418
179.13272898152744
168.7852870826208
162.26476096419265
162.26476096419265
152.44727677803883
